Tooling-Landscape/Unanimous-Understanding/Data Structures/Data.md

```diff
@@ -194,7 +204,7 @@ Standard-header := textfield
 
 ### Examples
 #### Example 1:
-Name: GPL-2.0 
+Name: GPL-2.0
 
 UniqueIdentifier: SHA1
```

Comment:
generally, "world" seems to go for SHA256 for everything (well, more or less). I am not seeing a technical collision risk here, but for the sake for being in line with others, it might be an idea to avoid SHA1 right from the beginning.